universal-application-tool-0.0.1/app/views/WebJarJsPaths.java

```diff
@@ -0,0 +1,30 @@
+package views;
+
+import java.util.Optional;
+
+/** Enum for holding the paths for web JARs needed throughout the code base. */
+public enum WebJarJsPaths {
```

Comment:
question (blocking): thoughts on using static constants instead of an enum? e.g.

```java
public final class WebJarJsPaths {
  public static final String AZURE_STORAGE_BLOB = "lib/azure__storage-blob/browser/azure-storage-blob.min.js";
}
```

Reply:
I like it I have updated the file